velox/functions/lib/KllSketch.h

```diff
@@ -88,6 +88,15 @@ struct KllSketch {
   std::vector<T, Allocator> estimateQuantiles(
       const folly::Range<Iter>& quantiles) const;
 
+  /// Estimate the values of the given quantiles.  This is more
+  /// efficient than calling estimateQuantile(double) repeatedly.
+  /// @tparam Iter Iterator type dereferenceable to double
+  /// @param quantiles Range of quantiles in [0, 1] to be estimated
+  /// @param out Pre-allocated memory to hold the result, must be at least as
+  ///  large as `quantiles`
+  template <typename Iter>
+  void estimateQuantiles(const folly::Range<Iter>& quantiles, T* out) const;
```

Comment:
Would it make sense to pass out by reference? This way it is guaranteed that it is not null.

Reply:
No it's pointing to the start of memory area to be written.  It's a common pattern to write multiple result.  I will add `FOLLY_NONNULL` to it.